tests/core/pyspec/eth2spec/test/merge/genesis/test_initialization.py

```diff
@@ -17,7 +19,7 @@ def eth1_init_data(eth1_block_hash, eth1_timestamp):
     }
 
 
-@with_merge_and_later
+@with_phases([MERGE])
```

Comment:
I think for the subsequent forks, the `execution_payload_header` field would be required?

Reply:
I personally think the transition logic might just stay in there, but we can debate it further down the line